osu.Game/Screens/Select/BeatmapDetails.cs

```diff
@@ -213,6 +212,19 @@ private void updateStatistics()
             displayMetrics(requestedBeatmap.Metrics, false);
         }
 
+        private void loadDetailsAsync(BeatmapInfo beatmap)
+        {
+            if (description == null || source == null || tags == null)
+                throw new InvalidOperationException($@"Requires all {nameof(MetadataSection)} elements to be non-null.");
+
+            Schedule(() =>

```

Comment:
this schedule is unnecessary

Reply:
I know - for some reason git added it back when I pushed my new changes...